presto-docs/src/main/sphinx/release/release-0.177.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.177
+=============
+
+General
+-------
+
+* Fix correctness issue when performing range comparisons over columns of type ``CHAR``.
+* Fix correctness issue due to mishandling of nulls and non-deterministic expressions in
+  inequality joins unless ``fast_inequality_join`` is disabled.
+* Fix excessive GC overhead caused by lambda expressions. There are still known GC issues
+  with captured lambda expressions pending fix.
+* Prevent duplicate columns in ``CREATE TABLE``.
+* Add support for null values on the left-hand side of a semijoin (i.e., ``IN`` predicate
+  with subqueries).
+* Add ``SHOW STATS`` to display table and query statistics.
+* Add support for preprocessing commands in the Presto CLI.  When the ``PRESTO_PREPROCESSOR``
+  environment variable is set, all commands are piped through the specified program before
+  being sent to the Presto server.
+* Improve implicit coercion support for functions involving lambda. Specifically, this fixes
+  unintuitive behavior on 2nd argument (initial state) of :func:`reduce` function.
+* Improve plans for queries involving ``ORDER BY`` and ``LIMIT`` by avoiding unnecessary
+  data exchanges.
+* Improve performance of queries containing window functions with identical ``PARTITION BY``
+  and ``ORDER BY`` clauses.
+* Improve performance of certain queries involving ``OUTER JOIN`` and aggregations, or
+  containing certain forms of correlated subqueries. This optimization is experimental
+  and can be turned on via the ``push_aggregation_through_join`` session property  and
+  ``optimizer.push-aggregation-through-join`` config option.
+* Improve performance of certain queries involving joins and aggregations.  This optimization
+  is experimental and can be turned on via the ``push_partial_aggregation_through_join``
+  session property.
+* Improve error message when lambda expression has a different number of arguments than expected.
+* Improve error message when certain invalid ``GROUP BY`` expressions containing lambda expressions.
+
+Hive
+----
+
+* Fix trailing space handling of CHAR type in RCFile reader.
+* Allow inserts into tables above partition per scan limits.
```

Comment:
This sentence reads funny. I'm not even sure what it's trying to convey. Is it something like "Allow inserts into tables containing more partitions than allowed during reads"?